features/minion/core/api/pom.xml

```diff
@@ -22,6 +22,7 @@
                         <Bundle-RequiredExecutionEnvironment>JavaSE-1.8</Bundle-RequiredExecutionEnvironment>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Bundle-Version>${project.version}</Bundle-Version>
+                        <Export-Package>org.opennms.minion.core.api.*;version="${project.version}"</Export-Package>
```

Comment:
I thought this was automatically exported. Is there a need to include this here?